LoopUI/Charts/PredictedGlucoseChart.swift

```diff
@@ -242,8 +218,8 @@ extension PredictedGlucoseChart {
     private func determineYAxisValues(axisLabelSettings: ChartLabelSettings? = nil) -> [ChartAxisValue] {
         let points = [
             glucosePoints, predictedGlucosePoints,
-            preMealOverridePoints, preMealOverrideDurationPoints,
-            targetGlucosePoints, targetOverridePoints,
+            preMealOverrideDurationPoints,
```

Comment:
This should also include the `targetOverrideDurationPoints` when determining the y-axis values. It doesn't have any affect unless those points would be off the displayed range of the chart (and in our use case they will always be in the display range). 

However, best to add it in case the behaviour of the charts change in the future.

Reply:
Good catch!!